docs/hugo/content/reference/_index.md

```diff
@@ -418,6 +418,14 @@ These resource(s) are available for use in the current release of ASO. Different
 
 To install the CRDs for these resources, your ASO configuration must include `insights.azure.com/*` as a one of the configured CRD patterns. See [CRD Management in ASO](https://azure.github.io/azure-service-operator/guide/crd-management/) for details on doing this for both [Helm](https://azure.github.io/azure-service-operator/guide/crd-management/#helm) and [YAML](https://azure.github.io/azure-service-operator/guide/crd-management/#yaml) based installations.
 
+### Next Release
+
+Development of these new resources is complete and they will be available in the next release of ASO.
+
+| Resource          | ARM Version        | CRD Version          | Supported From | Sample                                                                                                                                  |
+|-------------------|--------------------|----------------------|----------------|-----------------------------------------------------------------------------------------------------------------------------------------|
+| DiagnosticSetting | 2021-05-01-preview | v1api20210501preview | v2.10.0        | [View](https://github.com/Azure/azure-service-operator/tree/main/v2/samples/insights/v1api/v1api20210501preview_diagnosticsetting.yaml) |
```

Comment:
Unfortunately, we've just cut the v2.10 release of ASO (nearly two weeks late!). Change the `$supportedFrom` tag to `v2.11.0`.

Reply:
Updated to `v2.11.0`